src/core/sec.rs

```diff
@@ -74,9 +74,9 @@ enum AfInet {
 fn rdp_extended_infos(performance_flags: u32) -> Component {
     component![
         "clientAddressFamily" => U16::LE(AfInet::AfInet as u16),
-        "cbClientAddress" => DynOption::new(U16::LE(0), |x| MessageOption::Size("clientAddress".to_string(), x.inner() as usize + 2)),
+        "cbClientAddress" => U16::LE(2),
         "clientAddress" => b"\x00\x00".to_vec(),
-        "cbClientDir" => U16::LE(0),
+        "cbClientDir" => U16::LE(2),
```

Comment:
What's the 2? I think any time we use something that's non-zero we should use a constant or link to the spec so it's easy to look up.

Reply:
6df9004f2d4fbaafa0aebcfa98c70cd9cb2554a5